### ISSUE TITLE
Crash when closing the game

### ISSUE THREAD

**Issue by anbuc** [PunishedPineapple/Distance#7]
With the plugin enabled, the game chrashes on close

```
Stack trace:
   at System.Environment.get_StackTrace()
   at Dalamud.EntryPoint.VehCallback(IntPtr dumpPath, IntPtr logPath, IntPtr log) in C:\goatsoft\companysecrets\dalamud\EntryPoint.cs:line 77
   at ImGuiNET.ImGuiNative.igGetMousePos(Vector2* pOut)
   at ImGuiNET.ImGuiNative.igGetMousePos(Vector2* pOut)
   at Distance.PluginUI.UpdateAggroDistanceTextNode(DistanceInfo distanceInfo, Boolean show)
   at Distance.PluginUI.Dispose()
   at Distance.Plugin.Dispose()
   at Dalamud.Plugin.Internal.Types.LocalPlugin.<Dispose>b__53_0() in C:\goatsoft\companysecrets\dalamud\Plugin\Internal\Types\LocalPlugin.cs:line 246
   at Dalamud.Game.Framework.RunOnFrameworkThread(Action action) in C:\goatsoft\companysecrets\dalamud\Game\Framework.cs:line 142
   at Dalamud.Plugin.Internal.Types.LocalPlugin.Dispose() in C:\goatsoft\companysecrets\dalamud\Plugin\Internal\Types\LocalPlugin.cs:line 246
   at Dalamud.Utility.Util.ExplicitDisposeIgnoreExceptions[T](T obj, String logMessage, ModuleLog moduleLog) in C:\goatsoft\companysecrets\dalamud\Utility\Util.cs:line 551
   at Dalamud.Plugin.Internal.PluginManager.Dispose() in C:\goatsoft\companysecrets\dalamud\Plugin\Internal\PluginManager.cs:line 281
   at Dalamud.Service`1.Unset() in C:\goatsoft\companysecrets\dalamud\Service{T}.cs:line 190
   at System.RuntimeMethodHandle.InvokeMethod(Object target, Object[] arguments, Signature sig, Boolean constructor, Boolean wrapExceptions)
   at System.RuntimeType.InvokeMember(String name, BindingFlags bindingFlags, Binder binder, Object target, Object[] providedArgs, ParameterModifier[] modifiers, CultureInfo culture, String[] namedParams)
   at Dalamud.ServiceManager.UnloadAllServices() in C:\goatsoft\companysecrets\dalamud\ServiceManager.cs:line 236
   at Dalamud.Game.Framework.HandleFrameworkDestroy(IntPtr framework) in C:\goatsoft\companysecrets\dalamud\Game\Framework.cs:line 433
   at Dalamud.Game.Framework.HandleFrameworkUpdate(IntPtr framework) in C:\goatsoft\companysecrets\dalamud\Game\Framework.cs:line 412
```

[dalamud_appcrash.log](https://github.com/PunishedPineapple/Distance/files/9332041/dalamud_appcrash.log)

**Comment by PunishedPineapple** [comment]
Thank you for reporting this.  I'll get it fixed as soon as I can, but this upcoming week is kinda awful for me to get anything tested and released.  In the short term, you can either use a target besides mouseover for the aggro widget, or set the addon attachment for it to be something other than auto or screen space.  Sorry for the issue.

**Comment by anbuc** [comment]
No pressure, thanks for the workaround suggestion

**Comment by PunishedPineapple** [comment]
This should be fixed in version 1.0.0.2, which is available from the plugin installer.  Sorry again for taking so long to get a proper fix out for this!  If the problem somehow persists, please let me know.
